packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -159,6 +160,7 @@ import QuotaModal from 'web-pkg/src/components/Spaces/QuotaModal.vue'
 import {
   queryItemAsString,
   useAccessToken,
+  useCapabilityCreateUsersDisabled,
```

Comment:
Isn't it some sort of coding standard to use affirmative names (`useCapabilityCreateUsersEnabled`) instead of negative ones? I remember that in oc10 we paid attention to it.

Reply:
Code standart in web is: alignment to the backend